src/library/scala/Predef.scala

```diff
@@ -82,6 +82,12 @@ object Predef extends LowPriorityImplicits with DeprecatedPredef {
    */
   def classOf[T]: Class[T] = null // This is a stub method. The actual implementation is filled in by the compiler.
 
+  /**
+   * The `String` type in Scala has methods that come either from the underlying
+   * Java String (see the documentation corresponding to your Java version, for
+   * example [[http://docs.oracle.com/javase/7/docs/api/java/lang/String.html]]) or
+   * are added implicitly through [[scala.collection.immutable.StringOps]].
+   */
```

Comment:
There are a couple of JavaDoc-style comments still in Predef, but it shows Scala pride to use Scala-style, as at the top. It saves vertical space. Also, might as well link to Java 8 doc.